packages/neon_http_client/lib/src/neon_http_client.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:built_collection/built_collection.dart';
+import 'package:cookie_store/cookie_store.dart';
+import 'package:http/http.dart' as http;
+import 'package:meta/meta.dart';
+import 'package:neon_http_client/src/interceptors/interceptors.dart';
+import 'package:universal_io/io.dart';
+
+/// An exception caused by an error in a [NeonHttpClient].
+sealed class NeonClientException extends http.ClientException {
```

Comment:
```suggestion
sealed class NeonHttpClientException extends http.ClientException {
```
Since the class was renamed this should be renamed as well